bulk_grades/clients.py

```diff
@@ -46,6 +46,7 @@ def unchanged(self, data):
     dumps = loads = unchanged
 
 
+# pylint: disable=missing-docstring
```

Comment:
Maybe just add the docstring?

Reply:
I would but I don't really know what this does 😆